src/components/PokemonDisplay.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './pokemondisplay.css';
+
+export default function PokemonDisplay({ pokemon, type_1, type_2, attack, defense, url_image }) {
+  return (
+    <div className='pokemon-display'>
+      <h1>{pokemon}</h1>
+      <img src={url_image} />
+      <h3>Type: {type_1}{type_2}</h3>
```

Comment:
I might reccommend taking `{type_2}` out of this line, since you have type_2 already being displayed on its own line.